lib/analyzer/applications/node-modules-utils.ts

```diff
@@ -206,6 +208,17 @@ function groupFilesByDirectory(
   return filesByDir;
 }
 
+function isNodeAppFile(filepath: string): boolean {
+  const fileBase = path.basename(filepath);
+  return (
+    !filepath.includes("node_modules/") &&
+    (filepath.endsWith(".js") ||
```

Comment:
You can check all conditions excepts the first on `fileBase`

Reply:
Right, thanks